.circleci/config.yml

```diff
@@ -1,278 +1,57 @@
 version: 2
 jobs:
 
-  build-caches:
-    machine: true
+  lucene-and-others-javadoc:
+    docker:
+      - image: akmo03/groovy-jdk8:latest
```

Comment:
@romani Please let me know the details of the groovy image to checkstyle or your repo if pushed so that I can update hear